controllers/amphoracontroller_controller.go

```diff
@@ -254,6 +255,17 @@ func (r *OctaviaAmphoraControllerReconciler) reconcileNormal(ctx context.Context
 		return ctrl.Result{}, err
 	}
 
+	err = amphoracontrollers.EnsureAmphoraCerts(ctx, instance, helper, &Log)
```

Comment:
This function gets called for worker, housekeeping and healthmanager. It should suffice if only one of them calls them, because they all should share the same secret. I guess the same could be for `EnsureFlavors` and `EnsureLbMgmtNetworks` above. 

The different service deployments get created one after the other in `octavia_controller.go`. However, I am not sure if the reconcile loops of the different services can run concurrently. That might be a problem because `EnsureAmphoraCerts` is not reentrant.

Reply:
Handled in #216